gen/builders/generator.go

```diff
@@ -135,6 +141,11 @@ func (g *Generator) MessageVectorGroup(group string, vectors ...*MessageVectorGe
 				w = os.Stdout
 			} else {
 				file := filepath.Join(g.OutputPath, fmt.Sprintf("%s--%s.json", group, item.Metadata.ID))
+				// if file (probably) exists, skip if not force overwrite
+				if _, err := os.Stat(file); !os.IsNotExist(err) && !g.Overwrite {
```

Comment:
the most useful thing when re-generating vectors would be "i would like to re-generate vectors that are differing in a meaningful way." i wonder if there's a way to strip meta and have a mode for updating the files that are out of date.

Reply:
Yes agree, I should be able to trim out the meta and detect a change and only overwrite if there is changes.

Reply:
+1. I think three modes with proper logging of what's happening would make a great UX.

* `-f, --force`: forces the regeneration, whether or not the file exists.
* (no flag): only generates vectors which do not exist.
* `-c, --changed`: regenerates vectors that have changed (after stripping the _meta object) + vectors which do not exist.

We'd run the GitHub action with no flags, but have the other two for other operations.

I realise that `-f` is currently taken by the filter. I think we should change the filter flag to `-i, --include`, which is also a lot more explicit/intentional.

That would in turn free up `-f`, which we can then use for our GNU-style `force` switch.

Reply:
Argh, just seen this...I implemented `-r` for "replace" instead of `-f` and `-u` for "update" instead of `-c` - did not change `-f`.